service/controller/clusterapi/v31/resource/tccp/create.go

```diff
@@ -203,11 +208,9 @@ func (r *Resource) detachVolumes(ctx context.Context, cr v1alpha1.Cluster) error
 			return microerror.Mask(err)
 		}
 
-		// First shutdown the instances and wait for it to be stopped. Then detach
-		// the etcd and docker volume without forcing.
 		force := false
-		shutdown := true
-		wait := true
+		shutdown := false
+		wait := false
```

Comment:
Now that we stop the instance above we do not need to also do it in the EBS service.